src/sentry/api/serializers/models/groupsearchview.py

```diff
@@ -0,0 +1,29 @@
+from typing import TypedDict
+
+from sentry.api.serializers import Serializer, register
+from sentry.models.groupsearchview import GroupSearchView
+from sentry.models.savedsearch import SORT_LITERALS
+
+
+class View(TypedDict):
+    id: str
+    name: str
+    query: str
+    querySort: SORT_LITERALS
```

Comment:
Can this not be typed as `SortOptions`?

Reply:
I tried for like an hour to get this to work before just giving up. For some reason, mypy refuses to enforce the type if I set it to `SortOptions` - I set`querySort` of a `view` instance to `"awioef"` and mypy thought it was fine.